analytics.py

```diff
@@ -23,8 +24,17 @@ def __init__(self, _db):
         integrations = _db.query(ExternalIntegration).filter(ExternalIntegration.goal==ExternalIntegration.ANALYTICS_GOAL)
         # Turn each integration into an analytics provider.
         for integration in integrations:
+            kwargs = {}
+            module = integration.protocol
```

Comment:
I think I was the one who set this up originally and my reason for using the module as the protocol was that I thought it would be nice if someone could write a module and use it by adding it to the python path, without needing to get it merged into the main repo. We didn't follow this pattern for any other integrations, and now that we have the admin interface and docker, it doesn't make much sense. I think we should ditch this approach and make analytics work more like other integrations.

Reply:
I filed https://jira.nypl.org/browse/SIMPLY-1253 as a follow-up ticket.